sweep/sweeper.go

```diff
@@ -725,7 +725,17 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
 			inputs := s.updateSweeperInputs()
 
 			log.Debugf("Received new block: height=%v, attempt "+
-				"sweeping %d inputs", epoch.Height, len(inputs))
+				"sweeping %d inputs:\n%s", epoch.Height,
+				len(inputs), newLogClosure(func() string {
```

Comment:
I think the log closure doesn't need to be self-invoked. AFAIK that's handled by the logging library.

Reply:
fixed!